.github/workflows/ci-build.yml

```diff
@@ -168,4 +168,26 @@ jobs:
         fail_ci_if_error: false
         file: "manifest-updater/cover.out"
         flags: manifest_updater_tests
-        name: manifest-updater tests
\ No newline at end of file
+        name: manifest-updater tests
+
+  workflow-manager-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: workflow-manager
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.15
+    - name: Test
+      run: go test -race --coverprofile=cover.out --covermode=atomic ./... 
+    - name: Upload test coverage
+      if: success()
+      uses: codecov/codecov-action@v1
+      with:
+        fail_ci_if_error: false
+        file: "workflow-manager/cover.out"
+        flags: workflow_manager_tests
+        name: workflow-manager tests
```

Comment:
nit: no newline at EOF

Reply:
I really need to figure out what git setting I need to enable